entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -81,6 +87,8 @@ class EntityHandlerTests {
 
         every { entityService.exists(any()) } returns false
         every { entityService.createEntity(any()) } returns Entity(id = breedingServiceId, type = listOf("BeeHive"))
+        every { authorizationService.userIsCreator("mock-user") } returns true
+        every { authorizationService.createAdminLink(breedingServiceId, "mock-user") } just runs

```

Comment:
you could relax the mock and avoid this one, or (better) add a related `verify` clause.